javascript/javascript1/week2/homework/script.js

```diff
@@ -0,0 +1,143 @@
+function getFullName(firstName, lastName) {
+  const fullName = firstName + " " + lastName;
+  return fullName;
+}
+
+const fullName1 = getFullName("Anita", "Ngah");
+console.log(fullName1);
+
+function getFullNameFormel(
+  firstName,
+  lastName,
+  useFormel = false,
+  gender = "male"
+) {
+  if (!firstName || !lastName) {
+    return "pleaseProvideFullName";
+  }
+  let title = "";
+  if (useFormel) {
+    if (gender === "female") {
+      title = "lady";
+    } else {
+      title = "lord";
+    }
+  }
+  const fullName = title + " " + firstName + " " + lastName;
+  return fullName;
+}
+const fullName2 = getFullNameFormel("Anita", "Ngah", true, "female");
+console.log(fullName2);
+
+const fullName3 = getFullNameFormel("Valery", "Che", false, "male");
+console.log(fullName3);
+
+function getEventWeekday(daysFromToday) {
+  const weekdays = [
+    "Sunday",
+    "Monday",
+    "Tuesday",
+    "Wednesday",
+    "Thursday",
+    "Friday",
+    "Saturday",
+  ];
+  const today = new Date().getDay();
+  const eventDay = today + daysFromToday;
+  const exactEventDay = eventDay % 7;
+
+  return weekdays[exactEventDay];
+}
+// With todays weekday a wednesday
+console.log(getEventWeekday(8)); // Logs out "Thursday"
+
+// With todays weekday a wednesday
+console.log(getEventWeekday(2)); // Logs out "friday"
+
+function getIdeaOnWhatToWearToday(tempreture) {
+  if (tempreture < 0) {
+    return "its winter we can wear the coat and down jacket";
+  } else if (tempreture >= 0 && tempreture < 10) {
+    return "its winter we can wear the coat and pants and sweater";
+  } else if (tempreture >= 10 && tempreture <= 20) {
+    return "its autumm we can wear light coat and pants and huddies";
+  } else if (tempreture > 20 && tempreture <= 30) {
+    return "its summer we can wear tshirt and pants and socks";
+  } else {
+    return "its too hot we can wear shorts and tshirts and take lots of liquid";
+  }
+}
+console.log(getIdeaOnWhatToWearToday(20));
+console.log(getIdeaOnWhatToWearToday(20));
+
+let class07Student = [];
+function addNewStudent(studentName) {
+  if (studentName.trim() === "") {
+    console.log("can not add empty student name");
+    return;
+  }
+  if (class07Student.includes(studentName)) {
+    console.log(studentName + " is already in the class");
+    return;
+  }
+  if (studentName === "queen") {
+    class07Student.push(studentName);
+    console.log("queen has been added");
+    return;
+  }
+
+  if (class07Student.length >= 6) {
+    console.log("we are full.we cannot add more people to the class");
```

Comment:
Nit-pick - would be nice to keep logs clear and readable, even if they don't influence the code performance.
```suggestion
    console.log("We are full. We cannot add more people to the class.");
```